docker/docker-compose.local_dev.yml

```
@@ -12,26 +10,28 @@ services:
     build:
       context: prometheus
     environment:
-      # Linux:  http://localhost:8008
-      # MacOSX: http://host.docker.internal:8008
-      BEACON_URL: localhost:8008
-      VC_URL: localhost:5064
+      BEACON_URL: host.docker.internal:8008
+      VC_URL: host.docker.internal:5064
     restart: always
-    network_mode: host
     volumes:
       - "prometheus:/prometheus"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    ports:
+      - "9090:9090"
```

might be better to bind those to localhost

```suggestion
      - "127.0.0.1:9090:9090"
```